Tests/Functional/Scripts/requireAdditionalVendors.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+shopt -s extglob
+
+cd build/testproject/
+
+composer config extra.symfony.allow-contrib true
```

Comment:
line 5 is needed, otherwise, it runs the script inside the root.
line 7 seems to be needed, but just because otherwise you get prompted with "Do you want to execute this recipe?"

Reply:
Ok, fine for me